### PR TITLE
WildFly 2022 releases

### PR DESCRIPTION
**Pull request by bstansberry**

### CODE REVIEW

File: _posts/2022-01-21-WildFly-2022.adoc [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+title:  "WildFly Releases in 2022"
+date:   2022-01-20
+tags:   announcement roadmap
+author: bstansberry
+description: WildFly's release patterns will be changing in 2022
+---
+
+= WildFly Release Plans for 2022
+
+I'm my link:https://www.wildfly.org/news/2021/09/27/WildFly-Changes/[Changes are coming to WildFly] post last September, I tried to give a sense of how the transition to Jakarta EE 10 was likely to impact the next few WildFly releases. With WildFly 26 out the door and our efforts for 2022 ramping up, I want to give our community on update on how we see things playing out over the course of the year.
```
Comment (jamezp) [inline _posts/2022-01-21-WildFly-2022.adoc:12]: Should this be "In my"?

**Review by jamezp — COMMENTED** [review]
One minor comment

File: _posts/2022-01-21-WildFly-2022.adoc (hunk 1 above)
Comment (bstansberry) [reply _posts/2022-01-21-WildFly-2022.adoc:12]: Yep.

Comment (bstansberry) [reply _posts/2022-01-21-WildFly-2022.adoc]: Fixed

**Review by jamezp — APPROVED** [review]

**Pull request merged**